.github/actions/setup_env/action.yml

```diff
@@ -7,9 +7,11 @@ description:
   requirements, and the number of the week in the year, so that the cache
   refreshes weekly."
 inputs:
-  python-version:
```

Comment:
I removed this - it seems simpler to include the python version as a regular requirement.